### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by ladamski**
PR template

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<!-- Please add the WIP label if the PR isn't complete. -->
```
Comment (kdzwinel) [inline pull_request_template.md:1]: People can now create draft PRs so we should probably go with that rather than WIP label. (those have UI that prevents merging)

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+**Reviewer:**
```
Comment (kdzwinel) [inline pull_request_template.md:3]: We could suggest to look up privacy feature DRI first and, if that doesn't work, me and you second.

File: pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+Project task: 
+
+Relevant products which you have tested these changes against (i.e. extension, Windows, ...):
+- 
+
+Product PRs to test against:
+- 
+
+**Note: Upon merging this PR, you are responsible for updating the affected products to the latest version of this repo.**
```
Comment (kdzwinel) [inline pull_request_template.md:20]: I'm afraid that this will make sure that people don't contribute edge-cases etc. IMO all products should automatically pull and run changes and if test is correct (and this is up to person creating PR and feature DRI and/or us) then failing test on some platform is like a bug report for them to align.

File: pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+## Steps to test this PR:
```
Comment (kdzwinel) [inline pull_request_template.md:22]: Not sure if that applies? The only step is "run tests see if they pass". The bigger concern is if test is correctly describing the edge-case that we are trying to capture.

**Review by kdzwinel — COMMENTED** [review]
Great idea with a template. In fact I think we should have three: new feature, new test, new exception. Because instructions would be slightly different e.g. we should ask people adding exceptions to document them like https://github.com/duckduckgo/privacy-reference-tests/tree/main/https-upgrades#platform-exceptions.

I don't agree that it should be responsibility of a person adding test to ensure it passes on all platforms - it's just too steep of an ask and will kill contributions. Let's discuss on O-E meeting today.

File: pull_request_template.md (hunk 1 above)
Comment (ladamski) [reply pull_request_template.md:1]: 👍
